custom_example_rules/validate_migrations.py

```
@@ -0,0 +1,13 @@
+import subprocess
+
+from devrules.core.rules_engine import rule
+
+
+@rule(name="validate-migrations", description="Validate migrations run and are rollback safe")
+def validate_migrations(number: int) -> tuple[bool, str]:
+    try:
+        subprocess.run(["alembic", "upgrade", "head"], check=True)
```

Ruff (0.14.10)

Starting a process with a partial executable path

(S607)